src/types.ts

```diff
@@ -49,5 +55,7 @@ declare global {
 			config?: GuardianWindowConfig;
 		};
 		ga: UniversalAnalytics.ga;
+		// navigator must be readonly
```

Comment:
Minor - do we need this comment ? :)

Reply:
Yeah I suppose it's not all that useful anymore 😛